db/migrations/postgres/000002_create_data_table.up.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE data (
   datatype_version VARCHAR(64)     NOT NULL,
   hash             CHAR(64)        NOT NULL,
   created          BIGINT          NOT NULL,
-  value            BYTEA           NOT NULL,
+  value            TEXT           NOT NULL,
```

Comment:
nit: the `NOT NULL` on every column is now misaligned 😱

Reply:
![](https://c.tenor.com/VZvifZ0vcxkAAAAC/baker-freak-out.gif)

Reply:
oh, fixed ☺️